gr-soapy/python/soapy/bindings/soapy_common.cc

```diff
@@ -7,97 +7,29 @@
  *
  */
 
+#include "setting_string_conversion.h"
 #include "soapy_common.h"
 
 #include <SoapySDR/Types.hpp>
 #include <SoapySDR/Version.h>
 #include <stdexcept>
```

Comment:
Not used any more, right? Can be removed.

Reply:
This is needed because the functions were moved out of this file. They're still called into below.

Reply:
I mean the `stdexcept`; the code that needed that was moved to setting_string_conversion.h, but that doesn't include it (it does, indirectly through Soapy, though).

Reply:
Fixed